src/test/rustdoc/issue-27104.rs

```diff
@@ -16,5 +16,5 @@
 // @!has - 'extern crate std'
 // @!has - 'use std::prelude::'
 
-// @has - 'pub extern crate empty'
```

Comment:
I believe this is specifically testing `pub extern crate`, so this should probably stay.